ConceirgeDinning.Adapter.Zomato/Models/RestaurantDetails.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace ConceirgeDinning.Adapter.Zomato.Models
+{
+    public class RestaurantDetails
+    {
+        public Services R { get; set; }
```

Comment:
What is R?

Reply:
It is according to the contract provided by the supplier